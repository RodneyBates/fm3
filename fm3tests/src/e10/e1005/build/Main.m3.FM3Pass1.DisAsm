 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C449,L1(16_1),[25,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[26,2])
 ItkStmtListLt(L0(16_0),[26,2])
 ItkBlockDeclListLtPatch(C10,L11(16_b),[10,2])
 ItkBlockDeclListRt(L11(16_b),[10,2])
 ItkBlockDeclListSepPatch(C395,L10(16_a),[23,0])
 ItkConstDeclLtPatch(C395,[23,8])
 ItkConstDeclColonPatch(C401,[23,21],FALSE)
 ItkConstDeclEqualsPatch(C403,[23,21])
 ItkConstDeclRt([23,8])
 ItkIntLit(16_0(0),[23,23])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id5"Nonexistent",[23,8])
 ItkBlockDeclListSepPatch(C352,L9(16_9),[21,0])
 ItkConstDeclLtPatch(C352,[22,8])
 ItkConstDeclColonPatch(C358,[22,15],FALSE)
 ItkConstDeclEqualsPatch(C360,[22,15])
 ItkConstDeclRt([22,8])
 ItkIntLit(16_0(0),[22,17])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id6"Mumble",[22,8])
 ItkSkipLtPatch(C337,L2(16_2))
 ItkSkipRt(L2(16_2))
 ItkIntLit(16_0(0),[21,15])
 ItkDeclTypeAbsent()
 ItkBlockDeclListSepPatch(C294,L8(16_8),[19,0])
 ItkConstDeclLtPatch(C294,[20,8])
 ItkConstDeclColonPatch(C300,[20,14],FALSE)
 ItkConstDeclEqualsPatch(C302,[20,14])
 ItkConstDeclRt([20,8])
 ItkIntLit(16_0(0),[20,16])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id2"Lotsa",[20,8])
 ItkSkipLtPatch(C279,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkIntLit(16_0(0),[19,13])
 ItkDeclTypeAbsent()
 ItkBlockDeclListSepPatch(C245,L7(16_7),[17,0])
 ItkTypeDeclLtPatch(C245,[17,7])
 ItkTypeDeclEqualsPatch(C251,[17,11])
 ItkTypeDeclRt([17,7])
 ItkIdRefAtom(Id20"E2",[17,13])
 ItkDeclId(L9(16_9),Id19"Bad",[17,7])
 ItkBlockDeclListSepPatch(C211,L6(16_6),[16,0])
 ItkTypeDeclLtPatch(C211,[16,7])
 ItkTypeDeclEqualsPatch(C217,[16,12])
 ItkTypeDeclRt([16,7])
 ItkIdRefAtom(Id3"E1",[16,14])
 ItkDeclId(L9(16_9),Id18"Good",[16,7])
 ItkBlockDeclListSepPatch(C177,L5(16_5),[15,0])
 ItkTypeDeclLtPatch(C177,[15,7])
 ItkTypeDeclEqualsPatch(C183,[15,17])
 ItkTypeDeclRt([15,7])
 ItkIdRefAtom(Id5"Nonexistent",[15,19])
 ItkDeclId(L9(16_9),Id17"Worthless",[15,7])
 ItkBlockDeclListSepPatch(C140,L4(16_4),[14,0])
 ItkTypeDeclLtPatch(C140,[14,7])
 ItkTypeDeclEqualsPatch(C146,[14,14])
 ItkTypeDeclRt([14,7])
 ItkQualIdAtoms(Id15"NoIntf",Id16"Nada",[14,16],[14,25])
 ItkDeclId(L9(16_9),Id14"NoQual",[14,7])
 ItkBlockDeclListSepPatch(C106,L3(16_3),[13,0])
 ItkTypeDeclLtPatch(C106,[13,7])
 ItkTypeDeclEqualsPatch(C112,[13,14])
 ItkTypeDeclRt([13,7])
 ItkIdRefAtom(Id13"NoSuch",[13,16])
 ItkDeclId(L9(16_9),Id12"Undecl",[13,7])
 ItkBlockDeclListSepPatch(C69,L2(16_2),[12,0])
 ItkTypeDeclLtPatch(C69,[12,7])
 ItkTypeDeclEqualsPatch(C75,[12,20])
 ItkTypeDeclRt([12,7])
 ItkQualIdAtoms(Id4"Intf",Id11"Imppp",[12,22],[12,29])
 ItkDeclId(L9(16_9),Id10"IntfDotImppp",[12,7])
 ItkBlockDeclListSepPatch(C35,L1(16_1),[11,0])
 ItkTypeDeclLtPatch(C35,[11,7])
 ItkTypeDeclEqualsPatch(C41,[11,20])
 ItkTypeDeclRt([11,7])
 ItkQualIdAtoms(Id4"Intf",Id9"Yadda",[11,22],[11,29])
 ItkDeclId(L9(16_9),Id8"IntfDotYadda",[11,7])
 ItkTypeDeclLtPatch(C10,[10,7])
 ItkTypeDeclEqualsPatch(C16,[10,10])
 ItkTypeDeclRt([10,7])
 ItkIdRefAtom(Id4"Intf",[10,12])
 ItkDeclId(L9(16_9),Id7"T1",[10,7])
 ItkLookupScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
