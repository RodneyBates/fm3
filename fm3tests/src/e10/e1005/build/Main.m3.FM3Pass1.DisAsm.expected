 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L2(16_2),[1,0])
 ItkModuleBeginPatch(C551,L2(16_2),[25,2])
 ItkModuleRt(L2(16_2),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[26,2])
 ItkStmtListLt(L0(16_0),[26,2])
 ItkOpenDeclListLtPatch(C10,L11(16_b),[10,2])
 ItkOpenDeclListRt(L11(16_b),[10,2])
 ItkConstDeclLtPatch(C498,[23,8])
 ItkConstDeclRt([23,8])
 ItkIntLit(16_0(0),[23,23])
 ItkConstDeclValue([23,21])
 ItkDeclTypeAbsent([23,21])
 ItkConstDeclType([23,21])
 ItkDeclId(L9(16_9),Id5"Nonexistent",L-9223372036854775808(16_8000000000000000),[23,8])
 ItkOpenDeclListSep(L10(16_a),[10,2])
 ItkConstDeclLtPatch(C451,[22,8])
 ItkConstDeclRt([22,8])
 ItkIntLit(16_0(0),[22,17])
 ItkConstDeclValue([22,15])
 ItkDeclTypeAbsent([22,15])
 ItkConstDeclType([22,15])
 ItkDeclId(L9(16_9),Id6"Mumble",L-9223372036854775808(16_8000000000000000),[22,8])
 ItkOpenDeclListSep(L9(16_9),[10,2])
 ItkSkipLtPatch(C421,L2(16_2))
 ItkSkipRt(L2(16_2))
 ItkIntLit(16_0(0),[21,15])
 ItkConstDeclValue([21,13])
 ItkDeclTypeAbsent([21,13])
 ItkConstDeclType([21,13])
 ItkOpenDeclListSep(L9(16_9),[10,2])
 ItkConstDeclLtPatch(C374,[20,8])
 ItkConstDeclRt([20,8])
 ItkIntLit(16_0(0),[20,16])
 ItkConstDeclValue([20,14])
 ItkDeclTypeAbsent([20,14])
 ItkConstDeclType([20,14])
 ItkDeclId(L9(16_9),Id2"Lotsa",L-9223372036854775808(16_8000000000000000),[20,8])
 ItkOpenDeclListSep(L8(16_8),[10,2])
 ItkSkipLtPatch(C344,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkIntLit(16_0(0),[19,13])
 ItkConstDeclValue([19,11])
 ItkDeclTypeAbsent([19,11])
 ItkConstDeclType([19,11])
 ItkOpenDeclListSep(L8(16_8),[10,2])
 ItkTypeDeclLtPatch(C303,[17,7])
 ItkTypeDeclTypePatch(C318,[17,11])
 ItkTypeDeclRt([17,7])
 ItkIdRefAtom(Id20"E2",[17,13])
 ItkDeclId(L8(16_8),Id19"Bad",L-9223372036854775808(16_8000000000000000),[17,7])
 ItkOpenDeclListSep(L7(16_7),[10,2])
 ItkTypeDeclLtPatch(C262,[16,7])
 ItkTypeDeclTypePatch(C277,[16,12])
 ItkTypeDeclRt([16,7])
 ItkIdRefAtom(Id3"E1",[16,14])
 ItkDeclId(L8(16_8),Id18"Good",L-9223372036854775808(16_8000000000000000),[16,7])
 ItkOpenDeclListSep(L6(16_6),[10,2])
 ItkTypeDeclLtPatch(C221,[15,7])
 ItkTypeDeclTypePatch(C236,[15,17])
 ItkTypeDeclRt([15,7])
 ItkIdRefAtom(Id5"Nonexistent",[15,19])
 ItkDeclId(L8(16_8),Id17"Worthless",L-9223372036854775808(16_8000000000000000),[15,7])
 ItkOpenDeclListSep(L5(16_5),[10,2])
 ItkTypeDeclLtPatch(C177,[14,7])
 ItkTypeDeclTypePatch(C192,[14,14])
 ItkTypeDeclRt([14,7])
 ItkQualIdAtoms(Id15"NoIntf",Id16"Nada",[14,16],[14,25])
 ItkDeclId(L8(16_8),Id14"NoQual",L-9223372036854775808(16_8000000000000000),[14,7])
 ItkOpenDeclListSep(L4(16_4),[10,2])
 ItkTypeDeclLtPatch(C136,[13,7])
 ItkTypeDeclTypePatch(C151,[13,14])
 ItkTypeDeclRt([13,7])
 ItkIdRefAtom(Id13"NoSuch",[13,16])
 ItkDeclId(L8(16_8),Id12"Undecl",L-9223372036854775808(16_8000000000000000),[13,7])
 ItkOpenDeclListSep(L3(16_3),[10,2])
 ItkTypeDeclLtPatch(C92,[12,7])
 ItkTypeDeclTypePatch(C107,[12,20])
 ItkTypeDeclRt([12,7])
 ItkQualIdAtoms(Id4"Intf",Id11"Imppp",[12,22],[12,29])
 ItkDeclId(L8(16_8),Id10"IntfDotImppp",L-9223372036854775808(16_8000000000000000),[12,7])
 ItkOpenDeclListSep(L2(16_2),[10,2])
 ItkTypeDeclLtPatch(C49,[11,7])
 ItkTypeDeclTypePatch(C64,[11,20])
 ItkTypeDeclRt([11,7])
 ItkQualIdAtoms(Id4"Intf",Id9"Yadda",[11,22],[11,29])
 ItkDeclId(L8(16_8),Id8"IntfDotYadda",L-9223372036854775808(16_8000000000000000),[11,7])
 ItkOpenDeclListSep(L1(16_1),[10,2])
 ItkTypeDeclLtPatch(C10,[10,7])
 ItkTypeDeclTypePatch(C25,[10,10])
 ItkTypeDeclRt([10,7])
 ItkIdRefAtom(Id4"Intf",[10,12])
 ItkDeclId(L8(16_8),Id7"T1",L-9223372036854775808(16_8000000000000000),[10,7])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
